src/test/resources/gradle_plugin/wrapper/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
I am not a gradle(w) user but is it a good idea to have the wrapper checked in ATH source?